### ISSUE TITLE
Problem solved - re-installing Docker Desktop on Win 10 with WSL2 after Windows updates

### ISSUE THREAD

**Issue by huanfachen** [jreades/sds_env#4]
**TLDR**: this is not to report a problem but a note of the problem that I have met and solved. Think it may be a problem for other Windows users.

**Context**: the docker desktop on Win10 (with WSL2) suddenly collapsed after Windows updates. The software can't start, stopping after flashing.

**Process**: I spent nearly 10 hours to fix it - uninstalling and reinstalling. Although there were many posts and answers on docker.com and StackOverflow, most of them didn't work for my case.

**Error message** of re-installing Docker Desktop:
```
Component CommunityInstaller.EnableFeaturesAction failed:
em CommunityInstaller.InstallWorkflow.d__30.MoveNext()
— Fim do rastreamento de pilha do local anterior onde a exceção foi gerada —
em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
em CommunityInstaller.InstallWorkflow.d__24.MoveNext()
```

**Solution**:
The solution that solved my problem is [here](https://programmersought.com/article/29075200559/), although I am not 100% sure that many other things that I have attempted are helpful.

@jreades If anyone has a similar problem, I am more than happy to help. Just let me know.
